plugin/src/sidebar.ts

```diff
@@ -367,6 +368,39 @@ reearth.on("message", ({ action, payload }: PostMessageProps) => {
     }
   }
 
+  // CSV
+  if (action === "updatePointCSV") {
+    const { dataID, lng, lat, height } = payload;
+    const layerId = addedDatasets.find(ad => ad[0] === dataID)?.[2];
+    reearth.layers.override(layerId, {
+      data: {
+        csv: {
+          lngColumn: lng,
+          latColumn: lat,
+          heightColumn: height,
+        },
+      },
+    });
+  } else if (action === "resetPointCSV") {
+    const { dataID } = payload;
+    const layerId = addedDatasets.find(ad => ad[0] === dataID)?.[2];
+    reearth.layers.override(layerId, {
+      data: {
+        csv: undefined,
+      },
+    });
+  } else if (action === "getLocationNamesFromCSVFeatureProperty") {
+    const { dataID } = payload;
+    const layerId = addedDatasets.find(ad => ad[0] === dataID)?.[2];
+    const layer = reearth.layers.findById(layerId);
+    reearth.ui.postMessage({
+      action,
+      locationNames: getLocationNamesFromFeatureProperties({
+        ...(layer.computed?.features[0] || {}),
+      }),
+    });
+  }
```

Comment:
Can you comment this out too since it isn't being used yet (the useEffect with the ToDo comment seems to be the only thing using this)